### PR TITLE
H2IK.JoystickGremlin version 13.1.0

### PR DESCRIPTION
**Pull request by Amoenus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15635)

### CODE REVIEW

File: manifests/h/H2IK/JoystickGremlin/13.1.0/H2IK.JoystickGremlin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.1.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/h/H2IK/JoystickGremlin/13.1.0/H2IK.JoystickGremlin.installer.yaml:7]:
```suggestion
  Architecture: x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Both https://github.com/microsoft/winget-pkgs/pull/15632 and https://github.com/microsoft/winget-pkgs/pull/15635 seem to be reporting the same version as `13.1.0` in Control Panel. I'll check later if it's a mistake on my end.

I'll let https://github.com/microsoft/winget-pkgs/pull/15632 go through once you've made the changes and you can close this PR.

**Comment by Amoenus** [comment]
@ItzLevvie can confirm that 13.3.0 reports as 13.1.0 in Control Panel, seems like the author of the installer did not update it internally

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**